README.md

```diff
@@ -9,7 +9,11 @@ Covers project infrastructure initiatives for the OpenTelemetry project.
 * [Austin Parker](https://github.com/austinlparker), Honeycomb
 * [Trask Stalnaker](https://github.com/trask), Microsoft
 
+For more information about the maintainer role, see the [community repository](https://github.com/open-telemetry/community/blob/main/community-membership.md#maintainer).
```

Comment:
We should point to the [new link](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md#maintainer) itself instead of this one which says the document was moved.

The bot raised a similar PR for other SIGs as well 😕

Reply:
oh no, bad bot!

thanks for pointing it out, I'll have the bot send another wave, this should be easy to fix up